src/main/resources/static/downloads/gatling-demostore-api.postman_collection.json

```diff
@@ -0,0 +1,336 @@
+{
+	"info": {
+		"_postman_id": "97725589-7a64-4d38-b2ec-036d95a65ad9",
```

Comment:
AFAIK it's Postman's internal ID generated when creating the collection, which they include in the export file for reference; but it won't change anything when someone else imports the collection into their Postman install (because a new ID gets generated on import).

(The whole file is directly exported from Postman, no manual editing 🤷)